sorbet/rbi/shims/sorbet.rbi

```diff
@@ -0,0 +1,15 @@
+module T::Private
```

Comment:
Don't we have the runtime mock for this?

Reply:
`T::Private` exists in the exported RBIs but the types inside like `Final` or `Sealed` or `Abstract::Data` are not, so we need these for the static checker to not complain.